app/_hub/kong-inc/degraphql/_index.md

```diff
@@ -27,113 +26,127 @@ params:
   name: degraphql
   service_id: true
   dbless_compatible: 'yes'
-  config: null
+  config:
+    - name: graphql_server_path
+      required: true
+      default: "/graphql"
+      datatype: string
+      description: The path to the GraphQL server.
+      minimum_version: "3.0.x"
 ---
 ## Usage
 
-DeGraphQL needs a graphql endpoint to query. As an example, we are going to
-build a REST API around https://api.github.com GraphQL service. For that reason
-examples are going to include the header `Authorization: Bearer some-token`.
+DeGraphQL needs a GraphQL endpoint to query. As an example, we are going to
+build a REST API around the `https://api.github.com` GraphQL service. For that
+reason, the following examples include the header `Authorization: Bearer some-token`.
 
-Note this plugin differs from other plugins as far as configuration goes. It
-needs to be activated on a service that points to a GraphQL endpoint
-(sans `/graphql`).
+This plugin must be activated on a service that points to a GraphQL endpoint.
 
-### 1. Create a Service and a Route in Kong:
+### Create a Service and a Route
 
-  ```bash
-  $ curl -X POST http://localhost:8001/services \
-    --data name="github" \
-    --data url="https://api.github.com"
-  $ curl -X POST http://localhost:8001/services/github/routes \
-    --data paths="/api"
-  ```
+Create a Service and Route in {{site.base_gateway}}:
 
-### 2. Configure the Plugin on the Service
+```bash
+curl -X POST http://localhost:8001/services \
+  --data name="github" \
+  --data url="https://api.github.com"
 
-The plugin takes over the service. From this point on, the service represents
-our REST api and not the graphql endpoint itself. It will return a 404 Not Found
-if no DeGraphQL routes have been configured.
+curl -X POST http://localhost:8001/services/github/routes \
+  --data paths="/api"
+```
 
-  ```bash
-  $ curl -X POST http://localhost:8001/services/github/plugins \
-    --data name="degraphql"
-  ```
+### Configure the plugin on the Service
 
-### 3. Configure DeGraphQL Routes on the Service
+Set up the DeGraphQL plugin:
 
-Once the Plugin is activated on a Service, we can add our own routes to build
-our service, by defining URIs and associating them to GraphQL queries.
+```bash
+curl -X POST http://localhost:8001/services/github/plugins \
+  --data name="degraphql"
+```
 
-  ```bash
-  $ curl -X POST http://localhost:8001/services/github/degraphql/routes \
-    --data uri="/me" \
-    --data query="query { viewer { login } }"
+The plugin takes over the Service. From this point on, the Service represents
```

Comment:
> The plugin takes over the Service

I'm not sure what this is telling me. I _think_ it's something like:

```suggestion
Enabling the degraphql plugin will disable all existing routes in your service. From this point on, the Service represents
```

Reply:
More like this:
Enabling the degraphql plugin will disable the regular service, instead the plugin will now build the path and graphql query to hit the Graphql service with.